### ISSUE TITLE
Incorrect number of features for lppc function

### ISSUE THREAD

**Issue by jvdd** [fraunhoferportugal/tsfel#103]
Hey! I really like the plenty feature functions that are implemented in this library :smile: 
When toying around with your library, I found a bug (some unexpected behavior).

When calling `lppc` on some data with default arguments (i.e., `n_coeff`=12) it returns an array of length 13.
```python
from tsfel.feature_extraction import lpcc
# Calculate lpcc feature on some dummy data
len(lpcc(np.arange(500)))  # returns 13
```

However, when calling the `get_number_features` function, it returns that only 12 features will be returned.
```python
from tsfel import get_features_by_domain, get_number_features
# Get number of features for lpcc feature configuration
feat_dict = get_features_by_domain("spectral")
get_number_features({"dummy": {"dummy": sorted(feat_dict.values())[0]["LPCC"]}})  # returns 12
```

This difference is because the `get_number_features` function use the `n_coeff` parameter to determine the number of features. But in the `lpc` function the output is pre-pended with 1 value ([1]), making the output of `lpc` length 13 and thus also the output of `lpcc`.

So the number of features for `lppc` will always be `n_coeff` + 1. Is there a way to encode this in the feature configuration (dictionary)?

**Comment by mbarandas** [comment]
Hi @jvdd. Thanks for your feedback and for reporting this bug. 
We will address this issue in the next TSFEL release.

The `lpc` function should return the number of `n_coeff` given as input, not `n_coeff` + 1. Therefore, we will correct the `lpc` function and not encode this special case in the feature configuration dictionary.

**Comment by mbarandas** [comment]
Bug solved on PR #105
